solutions/devsprint-cecilia-saraiva-2/FinanceAppTests/Scenes/FinanceService/FinanceServiceTests.swift

```diff
@@ -27,53 +27,53 @@ final class FinanceServiceTests: XCTestCase {
     }
     
     override func tearDown() {
-        self.spy = nil
+        self.out = nil
         self.callCount = 0
         self.userProfileObject = nil
         self.url = nil
     }
 
-    func test_ShouldReturnUserProfileData() throws {
-        let (sut, networkStub) = makeSut(fileName: "user-profile", url: url, isRequestSuccess: true)
+    func test_ShouldReturnUserProfileData_WhenFetchUserProfileIsCalled() throws {
```

Comment:
[sugestao] e so a minha preferencia, eu colocaria o nome da função no início. Digamos que essa classe tem 10 funções, rapidamente eu procuro test_funcaoquetoprocurando_.... A pessoa pode buscar mais facilmente qual a função que quer, em vez de ler sobre o return antes de saber se esse teste lhe importa. Alem disso, ficaria um pouco menor tambem. Por exemplo:
```
func test_nameOfTheFunction_whenABCHappens_returnsXYZ
```
Novamente, essa e a minha preferencia e o que vejo as pessoas nas empresas onde trabalhei/trabalho seguirem.
Talvez nao tenha tanta importância agora porque temos poucos testes, mas quando tiver 10, 20, 30, pequenas coisas como o nome começam a importar mais. 😅

Reply:
Oie, faz super sentido, alterei 😄